frontend/src/components/DatabaseListSidePanel.vue

```diff
@@ -80,6 +83,29 @@ export default {
         });
     });
 
+    const kbarActions = computed((): Action[] => {
+      const actions = databaseListByEnvironment.value.flatMap((env: any) =>
+        env.childList.map((db: any) =>
+          createAction({
+            // `env.id` starts with "env", while `db.id` is a number
+            // so here `id` looks like "database.env.1234.5678"
+            id: `database.${env.id}.${db.id}`,
```

Comment:
1. db id is globally unique, so it's sufficient to use database.${db.id} or db.${db.id}

2. database and environment are both considered top level resources, so there is no need to embed them. A good place to look at our resource hierarchy is check our api ACL config https://github.com/bytebase/bytebase/blob/main/server/acl_casbin_policy_owner.csv

3. I would suggest to prefix with "bb." because a 3rd (like jira) may also has project concept and register to our kbar and we don't want have naming conflict.

Reply:
I do think "bb.navigation.home", "bb.project.${proj.id}", "bb.database.${db.id}" are nice. But for QuickActions "bb.quickaction.bb.instance.create" looks strange. Any suggestions?

Reply:
What prevent us from using bb.quickaction.instance.create instead of bb.quickaction.bb.instance.create ?

I would think quickaction is a separate submodule under bb which is different from bb.instance.xxx alike

Reply:
Use "bb." as a "root" namespace. Now we use these "primary" types below:
- bb.navigation.*
- bb.project.*
- bb.database.*
- bb.bookmark.*
- bb.quickaction.*

see [08929f06](https://github.com/bytebase/bytebase/pull/100/commits/08929f06922b8e6ce1299406e065bb3fd9847496)

Reply:
Yeah, I think bb.quickaction is a better idea.